packages/amplify-graphql-schema-generator/src/utils/vpc-helper.ts

```diff
@@ -0,0 +1,322 @@
+import {
+  RDSClient,
+  DescribeDBClustersCommand,
+  DescribeDBClustersCommandOutput,
+  DescribeDBClustersCommandInput,
+  DescribeDBInstancesCommand,
+  DescribeDBInstancesCommandOutput,
+  DescribeDBInstancesCommandInput,
+  DescribeDBSubnetGroupsCommand,
+} from "@aws-sdk/client-rds";
+import { 
+  IAMClient, 
+  CreateRoleCommand, 
+  GetRoleCommand, 
+  GetRoleCommandOutput, 
+  CreateRoleCommandOutput, 
+  Role,
+  CreatePolicyCommand,
+  Policy,
+  CreatePolicyCommandOutput,
+  AttachRolePolicyCommand,
+  waitUntilPolicyExists,
+  waitUntilRoleExists,
+} from "@aws-sdk/client-iam";
+import { 
+  LambdaClient, 
+  CreateFunctionCommand, 
+  CreateFunctionCommandInput, 
+  InvokeCommand, 
+  LogType, 
+  UpdateFunctionCodeCommand, 
+  UpdateFunctionCodeCommandInput, 
+  GetFunctionCommand,
+  GetFunctionCommandOutput,
+  waitUntilFunctionActive,
+  FunctionConfiguration,
+  DeleteFunctionCommand,
+} from "@aws-sdk/client-lambda"; 
+import * as fs from 'fs-extra';
+import ora from 'ora';
+import { printer } from "@aws-amplify/amplify-prompts";
+
+const DB_ENGINES = ["aurora-mysql", "mysql"];
+const spinner = ora('');
+
+export type VpcConfig = {
+  vpcId: string;
+  subnetIds: string[];
+  securityGroupIds: string[];
+};
+
+const checkHostInDBInstances = async (hostname: string, region: string): Promise<VpcConfig | undefined> => {
```

Comment:
A couple of things, but this file is missing function comments, and there are a few other places where it probably fails linting (double quotes mixed w/ single, etc.) Could you ensure this meets the lint standard for the project?

Reply:
Something wrong with my eslint plugin. I've updated this file now.